Rename from rstudioconnect to positconnect

ilyaZar opened the pull request:
Fix #888

rename add_rstudioconnect_file() to add_positconnect_file() in files:

- add_rstudio_files.R
- vignettes/c_deploy.Rmd
- tests/testthat/test-add_deploy_helpers.R
- inst/mantest/build.R
- inst/shinyexamle/dev/03_deploy.R
- _pkgdown.yml

Not touching NAMESPACE, .Rd-files, and NEWS.md as they are updated otherwise.

ColinFay commented:
Hey, 

Thanks. 

It think we should first Deprecate the functions before removing them, so that the change is not too brutal for people with old dev scripts :) 

So basically keeping something like : 

```
add_rstudioconnect_file <- function(...){
  .Deprecated("add_postitconnect_file")
  add_postitconnect_file(...)
}
```

For the old functions ?

ilyaZar commented:
yep! updated this

ilyaZar committed:
refactor: rename from rstudioconnect to positconnect

rename add_rstudioconnect_file() to add_positconnect_file() in files:

- add_rstudio_files.R
- vignettes/c_deploy.Rmd
- tests/testthat/test-add_deploy_helpers.R
- inst/mantest/build.R
- inst/shinyexamle/dev/03_deploy.R
- _pkgdown.yml

Not touching NAMESPACE, .Rd-files, and NEWS.md as they are updated otherwise.

Keep  add_rstudioconnect_file for compatibility but with depcrecation
note.

Refs: #888

VincentGuyader commented:
thks a lot
